cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -1970,21 +1973,12 @@ func (p *BicepProvider) ensureParameters(
 			return nil, fmt.Errorf("prompting for value: %w", err)
 		}
 
-		saveParameter, err := p.console.Confirm(ctx, input.ConsoleOptions{
-			Message:      "Save the value in the environment for future use",
-			DefaultValue: true,
-		})
```

Comment:
Why are we removing this? This could unintentionally lead to users saving/leaking secrets/passwords.

Reply:
discussed with the group, we will be removing this.